cherrypy/lib/cpstats.py

```diff
@@ -404,14 +415,17 @@ def record_stop(
 
 
 def locale_date(v):
```

Comment:
Note to self: this was first introduced in https://github.com/cherrypy/cherrypy/commit/d534fb34a175f92d47dbf5d99c9f9b41fbfb8539#diff-d17cf5948af6720a3bb009431606361a5dfe0696f9dba1e82e2f8e8db7e62081R391 and is never actually called within the framework. However, it seems to be intended to be used by the end-users who are supposed to mutate `StatsPage.formatting` (which is currently using `iso_format` there).